app/assets/stylesheets/application.scss

```diff
@@ -10,7 +10,11 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  */
-
+* {
```

Comment:
What is this for?

Reply:
biar the top navigation is on top of the side bar 😮